### DISCUSSION TITLE
Ideas to improve it

### DISCUSSION THREAD

**Discussion by EasyApplyBotUser** [voidbydefault/EasyApplyBot#27] [Ideas]
Here are some ideas to improve it. I think these will do good.

- Optional changing of the speed of applying.
- Sometimes it clicks a link that leads to a new tab, is there a way to prevent this?
- An option to make it run on startup. I plan to use this until I get a job.

It works great. Thank you!

**Comment by voidbydefault** [comment]
Thanks for your suggestions. 

> Optional changing of the speed of applying.

I will hardcode optimizations that I am kind-of aware of and testing before pushing for public use. I *won't* suggest over-optimization because I personally faced account restrictions twice. 

> Sometimes it clicks a link that leads to a new tab, is there a way to prevent this?

I fixed that already in last update, I haven't faced this in a long time. Are you sure using updated code? If possible, share screenshots (hide personal info) when this phenomenon gets triggers. Screenshots will help me identify patterns and update code accordingly.

> An option to make it run on startup. 

That's unfortunately not possible within the bot, but you may use Windows task schedule features to run Pycharm IDE then the bot script or otherwise open a command prompt with bot directory and then execute `python3 main.py`
